### PR TITLE
Order processor UI

### PR DESCRIPTION
**Pull request by TopETH**

### CODE REVIEW

File: src/components/Tables/OrderProcessorTable/index.tsx [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {(rowsPerPage > 0
+              ? data.slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
+              : data
+            ).map(
+              ({ orderId, extrinsicId, account, reward, timestamp }, index) => (
+                <StyledTableRow key={index}>
+                  <StyledTableCell align='center'>{orderId}</StyledTableCell>
```
Comment (Szegoo) [inline src/components/Tables/OrderProcessorTable/index.tsx:79]: This should be clickable. When clicked, it should open a modal that includes the order data (`<OrderCard/>`).

File: src/pages/processor.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+							sx={{ color: theme.palette.common.black }}
+						>
+							Order Processor UI
+						</Typography>
+						<Typography
+							variant='subtitle2'
+							sx={{ color: theme.palette.text.primary }}
+						>
+							Purpose: keep track of orders that got matched by whom, when and
+							at what price
```
Comment (Szegoo) [inline src/pages/processor.tsx:37]:
```suggestion
							See all the orders that were fulfilled
```

File: src/components/Tables/OrderProcessorTable/index.tsx (hunk 1 above)
Comment (TopETH) [reply src/components/Tables/OrderProcessorTable/index.tsx:79]: Okay, got it.

**Comment by Szegoo** [comment]
Could you resolve the conflict?

**Review by Szegoo — APPROVED** [review]

**Pull request merged**